code/index.html

```diff
@@ -3,28 +3,131 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat&display=swap" rel="stylesheet">
```

Comment:
I would maybe have added comments here `<!-- Link to Stylesheet -->` and `<!-- Link to Google Fonts -->` to make it easier to find for your future self.